src/components/helpers/factory.ts

```diff
@@ -0,0 +1,19 @@
+import { addCssToElement } from './css'
+
+export const createSpan = (
```

Comment:
It would be better to use a more [universal method make()](https://github.com/codex-team/editor.js/blob/next/src/components/dom.ts#L56) that we're using in other projects. One day we will create a separate package with all our dom utils.

Reply:
Should we use like this method in class or just a function? 
2) Do i just copy?

Reply:
just a function, you can copy I think